Add YAML description meta tags (#2582)
meilisearch/documentation: learn/fine_tuning_results/filtering.mdx

---
sidebarDepth: 4

> description: Filters allow your users to refine search results and are a fundamental element when creating faceted search interfaces.

Author: ```suggestion
description: Filters allow your users to refine search results. They can also be used when creating faceted search interfaces.
```
